spec/factories/users.rb

```diff
@@ -315,6 +315,17 @@
       ssn { '796043735' }
     end
 
+    factory :evss_user_no_participant, traits: [:loa3] do
```

Comment:
I won't block the PR on this, but I'd probably rewrite the spec so it didn't create a new factory, just doesn't seem necessary